src/users/users.controller.ts

```diff
@@ -8,12 +8,14 @@ import {
   Delete,
   ParseIntPipe,
   Put,
+  UseGuards,
 } from '@nestjs/common';
 import { UsersService } from './users.service';
 import { CreateUserDto } from './dto/create-user.dto';
 import { UpdatePartialUserDto } from './dto/update-partial-user.dto';
 import { IUser } from './interfaces/user.interface';
 import { UpdateUserDto } from './dto/update-user.dto';
+import { AccessTokenGuard } from 'src/guards/access-token.guard';
```

Comment:
this import may break your build, please be aware of such imports with 'src'
Depending on source directory it may ruin your build
So, please, every time check if it's not from src, but from '../../....'